tests/screenshot/apps/components/Popup.js

```diff
@@ -4,6 +4,14 @@ import React from 'react';
 const PopupTests = [
 	<Popup closeButton open>Popup!</Popup>,
 	<Popup centered open>Popup!</Popup>,
+	<Popup open duration="short">Popup!</Popup>,
```

Comment:
Isn't this the default value for the duration?

Reply:
Yes, it is. At first it was "medium", that is why the test is like this. I forgot to update it. Thanks!